uber/site_sections/registration.py

```diff
@@ -806,7 +806,7 @@ def new_checkin(self, session, message='', **params):
             if group:
                 session.match_to_group(attendee, group)
             attendee.checked_in = localized_now()
-            attendee.reg_station = cherrypy.session['reg_station']
+            attendee.reg_station = cherrypy.session.get('reg_station')
```

Comment:
FWIW, this one would be fine to leave as ``[]`` since we confirm that it's set above, but I'm okay with always using ``.get`` to prevent exceptions as a general practice.